DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
-# GOV.UK Notify Ruby client
+# Ruby client documentation
 
-This documentation is for developers interested in using this Ruby client to integrate their government service with GOV.UK Notify.
+This documentation is for developers interested in using the GOV.UK Notify Ruby client to send emails, text messages or letters.
 
-[![Gem Version](https://badge.fury.io/rb/notifications-ruby-client.svg)](https://badge.fury.io/rb/notifications-ruby-client)
+# Set up the client
 
-## Table of Contents
+## Install the client
 
-* [Installation](#installation)
-* [Getting started](#getting-started)
-* [Send messages](#send-messages)
-* [Get the status of one message](#get-the-status-of-one-message)
-* [Get the status of all messages](#get-the-status-of-all-messages)
-* [Get a template by ID](#get-a-template-by-id)
-* [Get a template by ID and version](#get-a-template-by-id-and-version)
-* [Get all templates](#get-all-templates)
-* [Generate a preview template](#generate-a-preview-template)
-* [Get received texts](#get-received-texts)
-
-## Installation
-
-Prior to usage an account must be created through the Notify admin console. This will allow access to the API credentials you application.
-
-You can then install the gem or require it in your application.
+Run the following code in the command line:
 
 ```
 gem install 'notifications-ruby-client'
 ```
 
-## Getting started
+Refer to the [client changelog](https://github.com/alphagov/notifications-ruby-client/blob/master/CHANGELOG.md) for the version number and the latest updates.
+
+## Create a new instance of the client
+
+Add this code to your application:
 
 ```ruby
 require 'notifications/client'
-client = Notifications::Client.new(api_key)
+client: Notifications::Client.new(api_key)
```

Comment:
This line doesn't need updating - it should still read `client = ...` instead of `client: ...`

(We do want to change `=>` to `:` in some cases, but because this isn't a Hash, we still need the `=` sign here.)